test/Atmos/EDMF/ekman_layer.jl

```diff
@@ -104,19 +106,21 @@ function main(::Type{FT}, cl_args) where {FT}
         N_quad,
         param_set,
         surface = NeutralDrySurfaceModel{FT}(param_set),
+        coupling = Coupled(),
```

Comment:
I would keep this decoupled for now. I would like this to test the Anelastic1D with grid mean viscosity